games-puzzle/quadra/quadra-1.3.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop
+
+DESCRIPTION="Tetris clone with multiplayer support"
+HOMEPAGE="https://github.com/quadra-game/quadra"
+SRC_URI="https://github.com/${PN}-game/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
fwiw
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI

On a related note, since I see github, had a look at the metadata and seems the remote-id is missing. Feel free to add it, or I could later (can keep the old google-code one along with it).

Reply:
i've added the github remote and remove google-code.